index.html

```diff
@@ -105,10 +110,11 @@ <h2 id="colored">I'm a Software Developer</h2>
  <div id="projects-section">
     <section id="intro">
       <h1>My Recent <br>Works</h1>
+    </section>
       <h2 id="line">
         <img src="images/ind.jpg" alt="_">
       </h2>
-    </section>
+    
```

Comment:
- [ **OPTIONAL** ] I think it would be better to improve your heading section styling, both for Mobile ad Desktop, to match the Figma template design by centering and giving the text the correct width, color, using the correct spacing(top, right, bottom, and left), styling your background image to cover the entire section without spacing at right, aligning everything correctly as from Figma template,...

- [ **OPTIONAL** ] **Make sure you apply the correct styling for the whole page as for now it doesn't the Figma design i.e. style correctly each section of your page to make your project sticks to the Figma design as much as possible.**

See one of the styling issues when changing the screen:
![heading_section](https://user-images.githubusercontent.com/83042742/173161914-e3b47ea5-729a-4bab-8414-3ecaeb775695.png)

Reply:
Hi, on my side its displaying well [check the welcome page here](https://microverse-students.slack.com/archives/D03JACHFNPL/p1654937212549829) and the contact form was now fixed [its here](https://microverse-students.slack.com/files/U03ELRQH6M8/F03K8HG96KD/image.png) and [check this too](https://www.loom.com/share/0e62af6daa7f469fa2fdf4ee417742de)